CHANGELOG.md

```diff
@@ -9,15 +9,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- ### Added -->
 
-<!-- ### Changed -->
+### Changed
+
+- [Auth] JWT now expects proper format -- `Bearer ...` -- [rfc6749](https://tools.ietf.org/html/rfc6749#section-7.1).
```

Comment:
what about

`- [Auth] No longer assumes the "Bearer" token type when no access token type is prepended in the Authorization header. Consequently, a client is expected to explicitly use "Bearer" for sending its JWT authorization token. More more details, see [RFC 6749](https://tools.ietf.org/html/rfc6749#section-7.1).`